ansible_collections/arista/avd/docs/contribution/style-guide.md

```diff
@@ -1,12 +1,26 @@
 # AVD Coding Style
 
-This page provides a list of guidelines to apply when developing the Jinja2 template in the AVD context. These rules apply for either creation or update of any Jinja2 file available in a `aristanetworks/ansible-avd` repository.
+This page provides a list of guidelines to apply when developing Python code or Jinja2 template in the AVD context. These rules apply for either creation or update of any Python or Jinja2 file available in a `aristanetworks/ansible-avd` repository.
 
 ## Python code style
 
 As AVD is an Ansible collection, we're required to follow guidelines from the official [Ansible documentation](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_best_practices.html) for all Python code.
 
-## YAML Syntax guidelines
+Furthermore the CI Pipeline (& pre-commit) for AVD enforces:
+
+- [Black](https://black.readthedocs.io/en/stable/index.html) version 22.8.0
+- [isort](https://pycqa.github.io/isort/) version 5.10.1
+- [Flake-8](https://flake8.pycqa.org/en/4.0.1/index.html) version 4.0.1
+- [pylint](https://pylint.pycqa.org/en/2.6/) version 2.6.0
+
+Configurations for above tools can be found in:
+
+- [pyproject.toml](../../../../../pyproject.toml)
+- [.pre-commit-config.yaml](../../../../../.pre-commit-config.yaml)
+- [.flake8](../../../../../.flake8)
+- [pylintrc](../../../../../pylintrc)
```

Comment:
we could probably move flake8 and pylintrc in pyproject,toml as well I think?

Reply:
I tried. Flake8 do not support pyproject.toml without a plugin.
Pylintrc is copied from Ansible, so I decided to leave it alone :)

Reply:
Ack - resolving this